specification/hugr.md

```diff
@@ -335,9 +364,22 @@ A **Predicate(T0, T1…TN)** type is an algebraic “sum of products” type,
 defined as `Sum(Tuple(#t0), Tuple(#t1), ...Tuple(#tn))` (see [type
 system](#type-system)), where `#ti` is the *i*th Row defining it.
 
-**TODO: update below diagram now that Conditional is “match”**
-
-<img src="attachments/2647818241/2647818344.png" width="1024px">
+```mermaid
+flowchart
```

Comment:
is there  a way to get the cases to appear side by side?

Reply:
I wanted to do that but couldn't find a way.

Reply:
oh sorry for not reading the PR description properly